### DISCUSSION TITLE
Goal tracking doesn't work with Clickhouse 22.6 on ARM64

### DISCUSSION THREAD

**Discussion by FarisZR** [plausible/analytics#2292] [Self-Hosted Support]
When using the suggested Clickhouse version in the hosting repo `22.6-alpine`
Goal tracking fails with the following error:
```
Uncaught (in promise) ApiError: ** (Clickhousex.Error) Code: 48. DB::Exception: multi-search all indices is not implemented when hyperscan is off (is it x86 processor?): while executing 'FUNCTION multiMatchAllIndices(pathname : 6, ['^/projects$', '^/en/projects$'] : 17) -> multiMatchAllIndices(pathname, ['^/projects$', '^/en/projects$']) Array(UInt64) : 18'. (NOT_IMPLEMENTED) (version 22.6.8.35 (official build))
```

Switching to `latest` fixes it, so I think the version need to be higher.
I also tested the non-alpine version, it had the same issue, so it's a version related issue.

Tested on ARM64.

**Comment by ruslandoga** [comment]
👋

~~Hm, interesting that `latest` works, I thought it wouldn't until [`hyperscan`](https://github.com/intel/hyperscan) adds arm64 support.~~

Ah, https://github.com/ClickHouse/ClickHouse/pull/38171

**Reply by ruslandoga** [reply]
I'll try out `latest` (or rather `22.9-alpine`) on my instance and open a PR into the hosting repo tomorrow.

**Reply by ruslandoga** [reply]
PR: https://github.com/plausible/hosting/pull/57
